pkg/config/constants/constants.go

```diff
@@ -67,9 +53,6 @@ const (
 
 	KubernetesGatewayName = "istio-autogenerated-k8s-gateway"
 
-	// IstioIngressNamespace is the namespace where Istio ingress controller is deployed
-	IstioIngressNamespace = "istio-system"
```

Comment:
We actually use this in our product because the control plane and ingress are in different namespaces

Reply:
You can still set a different value by `K8S_INGRESS_NS`

Reply:
Ah this is just a const; this is fine to remove then